CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ CHANGELOG
 
 _**Note:** This is in reverse chronological order, so newer entries are added to the top._
 
+Swift Next
+----------
+* [SE-0290][]:
+
+  It is now possible to write inverted availability statements by using the new `#unavailable` keyword:
```

Comment:
Repeating @rintaro's suggestion in a diff, since I think that gives you a button to apply the change: 

```suggestion
  It is now possible to write inverted availability conditions by using the new `#unavailable` keyword:
```